core/app/models/spree/credit_card.rb

```diff
@@ -2,6 +2,12 @@ module Spree
   class CreditCard < Spree::Base
     include ActiveMerchant::Billing::CreditCardMethods
 
+    if !ENV['DISABLE_DB_CONNECTION'] &&
```

Comment:
@caudatecoder can we rename it to `SPREE_DISABLE_DB_CONNECTION` ? thanks :)

Reply:
@caudatecoder it's the first instance of this ENV in the codebase

Reply:
@damianlegawiec oh, ok, it was in an unmerged PR :)
Will do then